scripts/regolith-sway-clamshell

```diff
@@ -1,14 +1,27 @@
 #!/usr/bin/env bash
+# This script takes one argument, which is the ACTION_TYPE
+#
+# ACTION_TYPE can be one of the following:
+#   LID_OPEN - The script is called when the lid is opened
+#   LID_CLOSE - The script is called when the lid is closed
+#
+# Requirements for using clamshell mode:
+#   - wm.clamshell.enabled is set to true
+#   - An inbuilt display is found
+#   - More than one display is connected
+
+ACTION_TYPE=$1
+
 
 exit_with_error() {
   echo "$1"
   echo "Exiting..."
   exit 1
 }
 
-get_action_cmd() {
+get_lid_close_cmd() {
   # Actions to take based on the value of wm.lidclose.action
-  local DEFAULT_LOCK="gtklock --background $(trawlcat regolith.lockscreen.wallpaper.file /dev/null)"
+  local DEFAULT_LOCK="/usr/bin/gtklock --background $(trawlcat regolith.lockscreen.wallpaper.file /dev/null)"
```

Comment:
question: why specify an absolute path to `/bin/local`?  (prevents users from building some updates in `/usr/local/bin`...)

Reply:
It doesn't have to be. I had a bug because of which gtklock was not getting called, and I thought using an absolute path could fix it.